docs/RationaleHW-SV.md

```diff
@@ -388,6 +388,28 @@ still many things in SystemVerilog that we cannot currently express in the SV
 dialect, this design makes it easy to incrementally build out new capabilities
 over time.
 
+#### Verbatim op
+The verbatim operation produces a typed value expressed by a string of
+SystemVerilog.  This can be used to access macros and other values that are
+only sensible as Verilog text.
```

Comment:
Please mention there are three ops here in the intro, the statement form the expr form and the se form

Reply:
Thanks, updated the text with the suggestions.